수박수박수박?.js

```diff
@@ -0,0 +1,9 @@
+function solution(n) {
+    let answer = "";
+    const textArr = ["수", "박"];
+    for (let i = 0; i < n; i++) {
```

Comment:
```suggestion
    const textArr = ['수', '박'];
    let answer = '';
    
    for (let i = 0; i < n; i++) {
```

let보다는 const로 선언한 변수를 먼저 써주시고 색깔이 다른 코드는 한줄 띄워주시는게 가독성에 더 좋아요. 그리고 자바스크립트에서 문자열을 지정할때 쌍따옴표보다는 홑따옴표를 쓰는것을 우선시합니다.